boards/arty-e21/src/main.rs

```diff
@@ -2,15 +2,15 @@
 
 #![no_std]
 #![no_main]
-#![feature(const_fn, in_band_lifetimes)]
+#![feature(const_fn, in_band_lifetimes, const_in_array_repeat_expressions)]
```

Comment:
Without this feature I was unable to write components for scheduler initialization that are generic over the number of processes a board supports. If a board did not want to use this feature, it could initialize the scheduler without using the component.

Reply:
```suggestion
#![feature(const_fn, in_band_lifetimes)]
// Required to support components that are generic over the number of processes
// a board supports. To avoid this feature, boards can initialize the scheduler directly
// without using the component interface.
//
// Tock expects this feature to stabilize in the medium future.
// Tracking: https://github.com/rust-lang/rust/issues/49147
#![feature(const_in_array_repeat_expressions)]
```

That makes sense. Maybe we should start a "documenting why / path to stable" on our features? I know this isn't the ideal place to prototype this out, but it's where it came to mind.

Reply:
this seems reasonable, but would we have to put that disclaimer in every main.rs?

Reply:
mhm, maybe we should centralize them somewhere instead, maybe a in the stabilization tracking issue